app/globals.css

```
@@ -148,6 +148,1131 @@
   --event-10k: oklch(0.68 0.181 44);
 }
 
+/* ------------------------------------------------------------------ */
+/*  칭호 이펙트 애니메이션                                              */
+/* ------------------------------------------------------------------ */
+
+@keyframes title-neon-pulse {
+  0%, 100% {
+    text-shadow: 0 0 4px currentColor, 0 0 10px currentColor, 0 0 20px currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)